lightning/src/ln/channelmanager.rs

```diff
@@ -5703,6 +5703,25 @@ impl<M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelManager<M, T, K, F
 		}
 	}
 
+	/// Gets inflight HTLC information by processing pending outbound `HTLCSource`s that are in
+	/// our channels.
```

Comment:
Yeah, that's correct. We'll work on an API for viewing pending payments in a separate PR, since we think it may need a different solution.

I'll add a little bit more context here to make its intended use more explicit.

Reply:
@valentinewallace I added a comment to specify that we use this information for pathfinding, but let me know if it can be better :)